canonical-websites/www.ubuntu.com
# Basic prometheus metrics on ports 9090-9099

- Enable prometheus metrics for Django workers on ports 9090 - 9099
- Upgrade the ./run script to 2.6.0 to facilitate testing

(You may want to help me out by also approving https://github.com/canonical-webteam/generator-canonical-webteam/pull/58)

QA
--

``` bash
./run exec --expose-port 8001 --expose-port 9090 --env DJANGO_DEBUG=false ./manage.py runserver --noreload 0.0.0.0:8001
```

- Go to http://0.0.0.0:9090 to see prometheus metrics
- Go to http://0.0.0.0:8001 and browse around the site a bit
- Reload http://0.0.0.0:9090 and see the numbers have gone up

Now you'll want to remove `DJANGO_DEBUG=false` from `.env.local` to reenable debug mode.
